ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -49,22 +81,112 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
           :host {
             @apply --layout-vertical;
           }
+          button {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin: 0;
+            padding: 0;
+            background: transparent;
+            border: none;
+            cursor: pointer;
+            text-decoration: none;
+            font-size: 1rem;
+          }
+          /* to fix a blinking default style on safari */
+          button:active {
+            color: inherit;
+          }
+          /* XXX - while we define our default focus state */
+          button:focus {
+            outline: none;
+          }
+          .heading {
+            width: calc(100% - 20px);
+            overflow-x: hidden;
+            text-align: start;
+          }
+          iron-icon {
+            --iron-icon-height: 20px;
+            --iron-icon-width: 20px;
+          }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 4px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          label {
+            cursor: pointer;
+            @apply --nuxeo-label;
+          }
+          .show-more-button a {
+            display: block;
+            margin-top: 4px;
           }
         </style>
 
-        <dom-repeat items="{{buckets}}">
+        <dom-if if="[[!collapsible]]">
           <template>
-            <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
-              [[item.label]] ([[item.docCount]])
-            </paper-checkbox>
+            <dom-if if="[[_isEmpty]]">
+              <template>
+                <label>[[label]]</label>
+                <span>[[i18n('checkboxAggregation.noResults')]]</span>
+              </template>
+            </dom-if>
+
+            <dom-if if="[[!_isEmpty]]">
+              <template>
+                <label>[[label]]</label>
+                <dom-repeat items="{{buckets}}">
```

Comment:
We are duplicating
```
<template>
  <label>[[label]]</label>
``` 
which we should be able to avoid by moving the `<dom-if if="[[!_isEmpty]]">` and `<dom-if if="[[_isEmpty]]">` to right after the span.